com.unity.netcode.gameobjects/Runtime/Transports/UNET/RelayTransport.cs

```diff
@@ -1,540 +0,0 @@
-#pragma warning disable 618 // disable is obsolete
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-using System;
-using System.Collections.Generic;
-using System.Net;
-using UnityEngine;
-using UnityEngine.Networking;
-
-namespace Unity.Netcode.Transports.UNET
-{
-    public static class RelayTransport
```

Comment:
did this ever work over the last year? :P

Reply:
Yes the relay transport is fully working and I know at least 10 people who are using it. If I find time I will move this into the contributions repository.